CMakeLists.txt

```diff
@@ -99,9 +99,23 @@ include_directories(
   "${LIB_INCLUDES}"
 )
 
+set(executables "")
+
+function(add_mvc_executable name)
+  set(executables
+    "source/${name}/View.cpp"
+    "source/${name}/Model.cpp"
+    "source/${name}/Controller.cpp"
+    PARENT_SCOPE
```

Comment:
Do we need `PARENT_SCOPE` ? If I understand correctly, it is only useful for a sub-cmake file to propagate a variable change in a parent cmakefile, but this is the root cmakefile so it has no parent ?

Reply:
Yes, we need it. When using the set function inside a CMake function, PARENT_SCOPE is necessary to modify the variable in the parent scope